src/main/java/net/sf/jabref/gui/FileExtensions.java

```diff
@@ -0,0 +1,34 @@
+package net.sf.jabref.gui;
+
+public enum FileExtensions {
+
+    //importatn: No dot before the extension
+    BIBTEX_DB(
+            "BibTeX Database", "bib"),
+    SILVER_PLATTERN(
+            "SilverPlatter", "dat"),
+    JSTYLE(
+            "Style File", "jstyle");
+
+    private final String[] extension;
+    private final String description;
+
+
+    private FileExtensions(String description, String... extension) {
+        this.extension = extension;
+        this.description = description;
+    }
+
+    public String[] getExtensions() {
```

Comment:
Normally I would agree with you, but in this case an Array is better, because a) varargs (...) as parameter is handled as array and  b) the array is then passed again as varags parameter to the FileNameFilterExtension.
A list would have to be converted back to array again to be passed as paramter.
And the decison for varargs was to support  multiple file extension for one type in the enum e.g. `Import_Format("Sample input format",xml,bib,dat)`
http://docs.oracle.com/javase/1.5.0/docs/guide/language/varargs.html